packages/braces/PKGBUILD

```diff
@@ -3,12 +3,12 @@
 
 pkgname=braces
 pkgver=0.4
-pkgrel=2
+pkgrel=3
 pkgdesc='A Bluetooth Tracking Utility.'
 url='http://braces.shmoo.com/'
 groups=('blackarch' 'blackarch-bluetooth')
 license=('GPL')
-depends=('php' 'mysql')
+depends=('php' 'mariadb')
 arch=('any')
 source=("http://braces.shmoo.com/blackhat-${pkgver//./}.tar.gz")
```

Comment:
The source is dead.

Reply:
https://github.com/BlackArch/blackarch/pull/3876/files

Reply:
Ok. So its removal will solve that conflicting issue anyway.